cx_Freeze/hooks.py

```diff
@@ -677,9 +677,10 @@ def load_win32com(finder, module):
 
 
 def load_win32file(finder, module):
-    """the win32api module implicitly loads the pywintypes module; make sure
-       this happens."""
+    """the win32api module implicitly loads the pywintypes and win32timezone
+       module; make sure this happens."""
     finder.IncludeModule("pywintypes")
+    finder.IncludeModule("win32timezone")
```

Comment:
This change is not needed as far as I can tell. I just tried with a new installation of Python 3.6 and it works just fine without win32timezone. Do you have an example that demonstrates the need?